api/src/pcapi/core/offers/validation.py

```diff
@@ -137,6 +137,25 @@ def check_stock_price(price: decimal.Decimal, offer: models.Offer, error_key: st
         )
         raise errors
 
+    if FeatureToggle.WIP_ENABLE_OFFER_PRICE_LIMITATION.is_active():
+        offer_price_limitation_rule = models.OfferPriceLimitationRule.query.filter(
+            models.OfferPriceLimitationRule.subcategoryId == offer.subcategoryId
+        ).one_or_none()
+        if offer_price_limitation_rule and offer.stocks:
+            reference_price = offer.lastValidationPrice or sorted(offer.stocks, key=lambda s: s.id)[0].price
```

Comment:
Attention ici, si `offer.lastValidationPrice` vaut `0`, cela signifie que l'offre a été approuvée en étant gratuite, mais on passe quand même à droite du `or`. C'est ce prix qui doit être pris en compte et non celui d'un stock, que l'on pourrait faire monter progressivement ainsi.
Suggestion, à confirmer/vérifier : 
```suggestion
            reference_price = offer.lastValidationPrice if offer.lastValidationPrice is not None else sorted(offer.stocks, key=lambda s: s.id)[0].price
```